tests/integration/targets/eos_vrf_global/tests/common/empty_config.yaml

```diff
@@ -0,0 +1,68 @@
+---
+- ansible.builtin.debug:
+    msg: START eos_vrf_global empty_config integration tests on connection={{ ansible_connection }}
+
+- name: Merged with empty configuration should give appropriate error message
+  register: result
+  ignore_errors: true
+  arista.eos.eos_vrf_global:
+    config:
+    state: merged
+
+- ansible.builtin.assert:
+    that:
+      - result.msg == 'value of config parameter must not be empty for state merged'
+
+- name: Replaced with empty configuration should give appropriate error message
+  register: result
+  ignore_errors: true
+  arista.eos.eos_vrf_global:
+    config:
+    state: replaced
+
+- ansible.builtin.assert:
+    that:
+      - result.msg == 'value of config parameter must not be empty for state replaced'
+
+- name: Overridden with empty configuration should give appropriate error message
+  register: result
+  ignore_errors: true
+  arista.eos.eos_vrf_global:
+    config:
+    state: overridden
+
+- ansible.builtin.assert:
+    that:
+      - result.msg == 'value of config parameter must not be empty for state overridden'
+
+- name: Rendered with empty configuration should give appropriate error message
+  register: result
+  ignore_errors: true
+  arista.eos.eos_vrf_global:
+    config:
+    state: rendered
+
+- ansible.builtin.assert:
+    that:
+      - result.msg == 'value of config parameter must not be empty for state rendered'
+
+- name: Parsed with empty configuration should give appropriate error message
+  register: result
+  ignore_errors: true
+  arista.eos.eos_vrf_global:
+    running_config:
+    state: parsed
+
+- ansible.builtin.assert:
+    that:
+      - result.msg == 'value of running_config parameter must not be empty for state parsed'
+
+- name: Purged with empty configuration should give appropriate error message
```

Comment:
what are we trying to assert with `purged`? I believe purged can work with empty config

Reply:
true, removing that test for purged, we dont need it!